NFC_EEPROM/mbed_app.json

```diff
@@ -1,8 +1,13 @@
 {
+  "macros": ["NFC_I2C_SDA=D14","NFC_I2C_SCL=D15","NFC_FIELD_DETECTION=D5"
```

Comment:
That is specific to a certain config; please add it to the K64F configuration and not the global configuration.